project1.css

```diff
@@ -1,46 +1,213 @@
 .navbar {
-	background-color: #7a7676;
+	background-color: #fc7b03;
 	display: flex;
+    top: 0;
+    left: 0;
+    width: 100%;
+    padding: 5px 0;	
+	position: fixed;
+	z-index: 2;
+	align: center;
+	text-align: center;
 	justify-content: center;
 }
 
-.nav-button {
-    display: inline-block;
+.navbar a {
     border: none;
-    padding: 1rem 2rem;
-    margin: 0;
+    padding: 1rem 1rem;
+	width: 20%;
     text-decoration: none;
-    background: #7a7676;
-    color: #ffffff;
-    font-family: Arial;
-    font-size: 1rem;
+    background: #fc7b03;
+    color: #000000;
+    font-family: "Andale Mono";
+	font-weight: bold;
     cursor: pointer;
-    text-align: center;
-    transition: background 250ms ease-in-out, 
-                transform 150ms ease;
-    -webkit-appearance: none;
-    -moz-appearance: none;
 }
 
-.nav-button:hover,
-.nav-button:focus {
-    background: #fc7b03;
+.navbar a:hover,
+.navbar a:focus {
+    background: #db6a00;
+}
+
+.intro-section {
+	font-family: "Andale Mono";
+	justify-content: center;
+	align-items: center;
+	padding-left: 200px;
```

Comment:
all the different parts of the website that have this padding should have it written in the same place, e.g.:
```css
.intro-section, .about-section, .portfolio-section {
   ...
}
```
that way if you decide to change it, it can be changed all at once.

also, there should be the same padding on the right because otherwise at a smaller screen size you get this:
![image](https://user-images.githubusercontent.com/25844355/150356256-df8d3162-4a24-4ee4-a09e-2218ffb00f6e.png)